### PR TITLE
Undefined name: BatchNorm2d() —> nn.BatchNorm2d()

### PR DESCRIPTION
**Pull request by cclauss**
Matches line 40.

flake8 testing of https://github.com/NVlabs/SPADE on Python 3.7.1

$ __flake8 . --count --select=E9,F63,F72,F82 --show-source --statistics__
```
./models/networks/normalization.py:80:36: F821 undefined name 'BatchNorm2d'
            self.param_free_norm = BatchNorm2d(norm_nc, affine=False)
                                   ^
1     F821 undefined name 'BatchNorm2d'
1
```

### CODE REVIEW

**Comment by taesungp** [comment]
Hi @cclauss 

thank you very much for pointing this out. We can't accept pull requests due to company policy but reflected your edit in 6d3aa2b.
